tests/TagFieldTest.php

```diff
@@ -413,6 +413,8 @@ public function testReadonlyTransformation()
         $field = new TagField('Tags', '', TagFieldTestBlogTag::get());
         $field->setTitleField('Title');
         $field->setValue(['Tag1']);
+        $form = new Form();
+        $field->setForm($form);
```

Comment:
Without this we get the following error due to calling `getSchemaState()` which we didn't used to call:

> LogicException: Field must be associated with a form to call Link(). Please use $field->setForm($form);